cmd/dep/root_analyzer.go

```diff
@@ -53,6 +63,58 @@ func (a *rootAnalyzer) InitializeRootManifestAndLock(dir string, pr gps.ProjectR
 	return
 }
 
+func (a *rootAnalyzer) cacheDeps(pr gps.ProjectRoot) error {
+	logger := a.ctx.Err
+	g, _ := errgroup.WithContext(context.TODO())
+	concurrency := 4
+
+	syncDep := func(pr gps.ProjectRoot, sm gps.SourceManager) error {
+		if err := sm.SyncSourceFor(gps.ProjectIdentifier{ProjectRoot: pr}); err != nil {
+			logger.Printf("Unable to cache %s - %s", pr, err)
+			return err
+		}
+		logger.Printf("Cached %s", pr)
```

Comment:
Looking at the output, I think it would be clearer if we cut down a bit on how much we are printing. Let's remove this print statement so that we are only logging when we start caching a package, and when the caching fails.

So the output would look more like this:
```
Caching package github.com/example/A
Caching package github.com/example/B
Caching package github.com/example/C
Successfully cached all deps.
```